src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSortVariant.unit.spec.jsx

```diff
@@ -386,11 +390,12 @@ describe('VAOS ProviderSortVariant on <CommunityCareProviderSelectionPage>', ()
     // When the user selects to sort providers by distance from a specific facility
     // Choose Provider based on facility address
     await screen.findByText(/Displaying 1 to /i);
-    fireEvent.change(await screen.getByLabelText('Show providers closest to'), {
-      target: {
-        value: '983',
-      },
+    const sel = await screen.findByTestId('providersSelect');
+    // call VaSelect custom event for onChange handling
+    sel.__events.vaSelect({
+      detail: { value: FACILITY_SORT_METHODS.distanceFromCurrentLocation },
```

Comment:
This should have the value of facility id
```suggestion
      detail: { value: '983' },
```